lib/rubocop/cop/lint/assignment_in_condition.rb

```diff
@@ -24,7 +24,10 @@ module Lint
       class AssignmentInCondition < Cop
         include SafeAssignment
 
-        MSG = 'Assignment in condition - you probably meant to use `==`.'.freeze
+        MSG = [
```

Comment:
Ah, k, so you're saying I need to check if safe assignment is enabled and tweak the error message accordingly?